tests/tck/features/lookup/EdgeIndexFullScan.feature

```diff
@@ -138,11 +98,90 @@ Feature: Lookup edge index full scan
-      | 4  | EdgeIndexFullScan | 0            |                                               |
-      | 0  | Start             |              |                                               |
+      | id | name      | dependencies | operator info |
+      | 3  | Project   | 4            |               |
+      | 4  | IndexScan | 0            |               |
+      | 0  | Start     |              |               |
+    # a IN b OR c
+    When profiling query:
+      """
+      LOOKUP ON edge_1
+      WHERE edge_1.col2_int IN [23 - 1 , 66/2] OR edge_1.col2_int==11
+      YIELD edge_1.col2_int
+      """
+    Then the result should be, in any order:
+      | SrcVID | DstVID | Ranking | edge_1.col2_int |
+      | "101"  | "102"  | 0       | 11              |
+      | "102"  | "103"  | 0       | 22              |
+      | "103"  | "101"  | 0       | 33              |
+    And the execution plan should be:
+      | id | name      | dependencies | operator info |
+      | 3  | Project   | 4            |               |
+      | 4  | IndexScan | 0            |               |
+      | 0  | Start     |              |               |
+    # a IN b OR c IN d
+    When profiling query:
+      """
+      LOOKUP ON edge_1
+      WHERE edge_1.col2_int IN [23 - 1 , 66/2] OR edge_1.col1_str IN [toUpper("r")+"ed1"]
+      YIELD edge_1.col1_str, edge_1.col2_int
+      """
+    Then the result should be, in any order:
+      | SrcVID | DstVID | Ranking | edge_1.col1_str | edge_1.col2_int |
+      | "101"  | "102"  | 0       | "Red1"          | 11              |
+      | "102"  | "103"  | 0       | "Yellow"        | 22              |
+      | "103"  | "101"  | 0       | "Blue"          | 33              |
+    And the execution plan should be:
+      | id | name      | dependencies | operator info |
+      | 3  | Project   | 4            |               |
+      | 4  | IndexScan | 0            |               |
+      | 0  | Start     |              |               |
+    # a IN b AND c (EdgeIndexPrefixScan)
+    When profiling query:
+      """
+      LOOKUP ON edge_1
+      WHERE edge_1.col2_int IN [11 , 66/2] AND edge_1.col2_int==11
+      YIELD edge_1.col2_int
+      """
+    Then the result should be, in any order:
+      | SrcVID | DstVID | Ranking | edge_1.col2_int |
+      | "101"  | "102"  | 0       | 11              |
+    And the execution plan should be:
+      | id | name                | dependencies | operator info |
+      | 3  | Project             | 4            |               |
+      | 4  | EdgeIndexPrefixScan | 0            |               |
+      | 0  | Start               |              |               |
+    # a IN b AND c IN d
+    # List has only 1 element, so prefixScan is applied
+    When profiling query:
+      """
+      LOOKUP ON edge_1
+      WHERE edge_1.col2_int IN [11 , 66/2] AND edge_1.col1_str IN [toUpper("r")+"ed1"]
+      YIELD edge_1.col1_str, edge_1.col2_int
```

Comment:
test following scenarios:
1. NOT IN
2. col1 has index and col2 also has, col1 IN [] and col2 IN []
3. col1 has index and col2 doest not have, col1 IN [] and col2 IN []
4. col1,col2 has composite index, col1 IN [] and col2 IN []

Reply:
Cases have been added.